pymc/distributions/discrete.py

```diff
@@ -570,6 +576,11 @@ def dist(cls, mu, *args, **kwargs):
         # mode = intX(at.floor(mu))
```

Comment:
Seems like we have a leftover from before. Do you mind removing the commented out code?
```suggestion
```

Reply:
Also note that we used the floor of mu, we should do the same in the new `get_moment`

Reply:
Is the `IntX` necessary or should I just include the `at.floor`?

Reply:
Not needed either I think